data-mongodb/src/main/java/io/micronaut/data/mongodb/transaction/MongoReactiveTransactionManagerFactory.java

```diff
@@ -65,15 +66,15 @@ <T> SynchronousFromReactiveTransactionManager<T> buildSynchronousTransactionMana
     @Singleton
     <T> AsyncUsingReactiveTransactionOperations<T> buildPrimaryAsyncTransactionOperations(@Primary ReactorReactiveTransactionOperations<T> reactiveTransactionOperations,
                                                                                           @Nullable CoroutineTxHelper coroutineTxHelper,
-                                                                                          @Named(TaskExecutors.IO) ExecutorService executorService) {
-        return new AsyncUsingReactiveTransactionOperations<>(reactiveTransactionOperations, coroutineTxHelper);
+                                                                                          @Nullable ReactorCoroutineTxHelper reactiveHibernateHelper) {
+        return new AsyncUsingReactiveTransactionOperations<>(reactiveTransactionOperations, coroutineTxHelper, reactiveHibernateHelper);
```

Comment:
Not important probably, but naming `reactiveHibernateHelper` in Mongo transaction factory could be confusing (same as in the constructor below)